guava/src/com/google/common/collect/Ordering.java

```diff
@@ -491,7 +491,7 @@ protected Ordering() {}
    * @param comparators the comparators to try in order
    */
   @GwtCompatible(serializable = true)
-  public static <T> Ordering<T> compound(Iterable<? extends Comparator<? super T>> comparators) {
+  public static <T extends @Nullable Object > Ordering<T> compound(Iterable<? extends Comparator<? super T>> comparators) {
```

Comment:
Stray space within `Object >`.  But, you don't need this change since it's the default.
Omitting changes like this one will keep the diffs smaller.
You may also be able to remove some `import ...Nullable;` statements as a result, and avoid some line breaks.